Business-card/READme.md

```diff
@@ -0,0 +1,10 @@
+# Business card
+Font: Raleway, via google fonts
+
+Colors:
+
+- pigment green: #139D3D
+- spring bud: #A5F200
+- phthalo blue: #020887
```

Comment:
This information is the same in most new branding, so it should probably be in the main README file

Reply:
How about prints-and-digital? It represents the content and purpose of the assets in the folder

Reply:
I am not sure to understand what you mean here

Reply:
we could have `prints-and-digital` for all the media assets; stickers, flyers, business cards, merchandise and letterheads.

Reply:
Sounds good. I was not sure I followed because you answered here to my comment on the README file 😄

Reply:
😄sorry about that.

Reply:
should all the uploads to `prints-and-digital` be in one PR?

Reply:
no it can be in different PRs. But please apply the same to the other PRs. Thanks